classes/functions/FixedDateFunction.php

```diff
@@ -51,6 +51,6 @@ public function getCallable()
 
     public function increment($increment)
     {
-        $this->timestamp += $increment;
+        $this->timestamp += (int) $increment;
```

Comment:
timestamp is an int and using e.g. float as $increment would implicitly cast it to int anyways, which was a deprecation warning.